.travis.yml

```diff
@@ -1,21 +1,21 @@
 language: php
 
 php:
-  - "5.5"
+  - "7.0"
```

Comment:
IIRC, @arschmitz used to say that there were some issues in upgrading to PHP 7.0.